graylog2-server/src/test/java/org/graylog2/rest/resources/users/UsersResourceTest.java

```diff
@@ -137,6 +139,7 @@ public void setUp() {
         usersResource = new TestUsersResource(userManagementService, paginatedUserService, accessTokenService,
                 roleService, sessionService, new HttpConfiguration(), subject,
                 sessionTerminationService, securityManager, globalAuthServiceConfig, clusterConfigService, userService);
+        when(clusterConfigService.get(PasswordComplexityConfig.class)).thenReturn(PasswordComplexityConfig.DEFAULT);
```

Comment:
While checking, the `getOrDefault()` is used:
```suggestion
        when(clusterConfigService.getOrDefault(PasswordComplexityConfig.class, PasswordComplexityConfig.DEFAULT)).thenReturn(PasswordComplexityConfig.DEFAULT);
```